packages/mcp-server-supabase/src/server.test.ts

```diff
@@ -381,7 +381,7 @@ describe('tools', () => {
     });
   });
 
-  test('create project chooses closest region when undefined', async () => {
+  test('create project without region fails', async () => {
```

Comment:
I modified the old test but could potentially remove the test since the schema ensures this param is provided and we don't test like this in other cases